### PR TITLE
846: Create DNS for new stand alone test trusted directory and update relevant configs

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Create new variable in postman and use for any endpoint that calls /jwkms

Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/846

### CODE REVIEW

File: postman/v1.0/Secure API Gateway v1.0 - Open Banking UK TPP Flows.postman_collection.json [hunk 1]
```diff
@@ -281,7 +281,7 @@
 									}
 								},
 								"url": {
-									"raw": "https://{{IG-FQDN}}/jwkms/apiclient/issuecert",
+									"raw": "https://{{TRUSTEDDIR-FQDN}}/jwkms/apiclient/issuecert",
 									"protocol": "https",
 									"host": [
 										"{{IG-FQDN}}"
```
Comment (dbadham-fr) [inline postman/v1.0/Secure API Gateway v1.0 - Open Banking UK TPP Flows.postman_collection.json:287]: I made this mistake before, I don't really understand the purpose of the "raw" field as the "host" below needs to be updated too, seems to duplicate data for no obvious reason.

**Review by dbadham-fr — CHANGES_REQUESTED** [review]

File: postman/v1.0/Secure API Gateway v1.0 - Open Banking UK TPP Flows.postman_collection.json (hunk 1 above)
Comment (shaunharrisonFR) [reply postman/v1.0/Secure API Gateway v1.0 - Open Banking UK TPP Flows.postman_collection.json]: That is done now, thank you!

**Review by dbadham-fr — CHANGES_REQUESTED** [review]
Please revert the changes to the OB collections, we only want to change the Secure API Gateway - Plain FAPI collection (v3) as we will only support this change in core.

The change to the environment is correct.

**Comment by shaunharrisonFR** [comment]
> Please revert the changes to the OB collections, we only want to change the Secure API Gateway - Plain FAPI collection (v3) as we will only support this change in core.
> 
> The change to the environment is correct.

This is done, github is showing a change in the desc for v1 but can't see an actual difference

**Review by dbadham-fr — APPROVED** [review]

**Pull request merged**